code/modules/antagonists/traitor/contractor/datums/syndicate_contract.dm

```diff
@@ -518,14 +518,52 @@
 
 	QDEL_LIST(temp_objs)
 
-	// Chance for souvenir or bruises
+	// Injuries due to questioning and souvenirs
 	if(prob(RETURN_SOUVENIR_CHANCE))
 		to_chat(M, "<span class='notice'>Your captors left you a souvenir for your troubles!</span>")
 		var/obj/item/souvenir = pick(souvenirs)
 		new souvenir(closet)
-	else if(prob(RETURN_BRUISE_CHANCE) && M.health >= 50)
-		to_chat(M, "<span class='warning'>You were roughed up a little by your captors before being sent back!</span>")
-		M.adjustBruteLoss(RETURN_BRUISE_DAMAGE)
+	if(prob(RETURN_INJURY_CHANCE) && M.health >= 50)
+		var/obj/item/organ/external/injury_target
+		var/nolimb = FALSE
+		if(prob(20)) //remove a limb
+			if(prob(50))
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+				if(!injury_target)
+					nolimb = TRUE
+					return
+				injury_target.droplimb()
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! Oh god something's missing!</span>")
+		else //fracture
+			if(ismachineperson(M))
+				M.emp_act(1)
+				M.adjustBrainLoss(30)
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like some of your componenets are loose!</span>")
+
+			if(isslimeperson(M))
+				injury_target = M.get_organ(pick(BODY_ZONE_PRECISE_R_HAND, BODY_ZONE_PRECISE_L_HAND, BODY_ZONE_PRECISE_R_FOOT, BODY_ZONE_PRECISE_L_FOOT))
+				if(!injury_target)
+					nolimb = TRUE
+					return
+				injury_target.cause_internal_bleeding()
+
+				injury_target = M.get_organ(BODY_ZONE_CHEST)
+				injury_target.cause_internal_bleeding()
+				to_chat(M, "<span class='warning'>You were interrogated by your captors before being sent back! You feel like your inner membrane has been punctured!</span>")
+
+			else if(prob(10))
```

Comment:
This can also proc for an IPC currently because you have no `else` before the slime `if`